test/rubocop/cop/minitest/multiple_assertions_test.rb

```diff
@@ -15,6 +15,14 @@ def test_asserts_twice
           assert_equal(foo, bar)
           assert_empty(array)
         end
+
+        def test_asserts_three_times
```

Comment:
Can you separate this new test example from existing `def test_registers_offense_when_multiple_expectations` (L10) to a new test method?

Reply:
Done.